INSTALL.md

```diff
@@ -103,8 +108,257 @@ docker container stop elasticsearch_test; docker container rm elasticsearch_test
 ```
 
 The following command starts a local MongoDB instance, runs the test suite, then stops and deletes the containers:
+
 ```shell
 docker run -d --name mongo_test -p 27017:27017 -d mongo:4.4.6 \
 && OPTIMADE_DATABASE_BACKEND="mongodb" py.test; \
 docker container stop mongo_test; docker container rm mongo_test
 ```
+
+## Container image
+
+### Retrieve the image
+
+The [`optimade` container image](https://github.com/Materials-Consortia/optimade-python-tools/pkgs/container/optimade) is available from the [GitHub Container registry](https://ghcr.io).
+To pull the latest version using Docker run the following:
+
+```shell
+docker pull ghcr.io/materials-consortia/optimade:latest
+```
+
+!!! note
+    The `:latest` part can be left out, as the `latest` version will be pulled by default.
+
+If you'd like to pull a specific version, this can be done by replacing `latest` in the command above with the version of choice, e.g., `0.17.1`.
+To see which versions are available, please go [here](https://github.com/Materials-Consortia/optimade-python-tools/pkgs/container/optimade/versions).
+
+### Run a container
+
+When starting a container from the image there are a few choices.
+It is possible to run both the standard OPTIMADE server implemented in this repository as well as an index meta-database from this image.
+The key is setting the environment variable `MAIN`.
+
+| **MAIN** | **Result** |
+|:---:|:--- |
+| `main` | Standard OPTIMADE server. |
+| `main_index` | Index meta-database OPTIMADE server. |
+
+Using Docker, the following command will run a container from the image:
+
+```shell
+# rm will remove container when it exits.
+# detach will run the server in the background.
+# publish will run the server from the host port 8080.
+# name will give the container a handy name for referencing later.
+docker run \
+    --rm \
+    --detach \
+    --publish 8080:5000 \
+    --env MAIN=main \
+    --name my-optimade \
+    ghcr.io/materials-consortia/optimade:latest
+```
+
+The server should now be available at [localhost:8080](http://localhost:8080).
+
+### Configure the server
+
+The server will use test data and a test configuration by default.
```

Comment:
```suggestion
By default, the server will use the test configuration that includes test data for structures and references and an in-memory mongomock database backend.
```

Reply:
Changed the wording slightly, since it sounded like the test data was part of the configuration, which is not true.